src/main/java/com/watch/shop/app/service/WatchServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.watch.shop.app.service;
+
+import com.watch.shop.app.model.Watch;
+import com.watch.shop.app.model.WatchStore;
+
+import java.util.Comparator;
+import java.util.List;
+
+import static java.util.Objects.isNull;
+
+public class WatchServiceImpl implements WatchService {
+
+    private final WatchStore watchStore = WatchStore.getInstance();
+
+    public List<Watch> getWatches() {
+        return watchStore.getWatches();
```

Comment:
please use unmodifiable collection there, we need to be sure, that in chains of method calls no collection modification will be allowed